index.js

```diff
@@ -25,11 +36,19 @@ module.exports = promisedHandlebars
  *   the promised results. This placeholder may not occur in the template or any partial. Neither
  *   my any helper generate the placeholder in to the result. Errors or wrong replacement will
  *   happen otherwise.
+ * @param {Promise} options.Promise the promise implementation to use. Defaults to global.Promise
```

Comment:
I would have made `promise` lowercase, but since it is a constructor, it may also be uppercase. We'll leave it like that.